pkg/utils/jwt.go

```diff
@@ -0,0 +1,33 @@
+package utils
+
+import (
+	"fmt"
+	"github.com/golang-jwt/jwt/v4"
+)
+
+func GetFieldFromJWT[T any](token string, field string) (T, error) {
```

Comment:
Seems that the generic type is only ever string - so it feels that this could just as well be non-generic until needed?

Reply:
I could go either way, since a function that only returns strings here would break if someone requested a differently typed field.

I think what I'll do is rename this to `GetStringFieldFromJWT` and remove the parameterized type. That way it's explicit.